solr/config/configsets/all_collection_oag56_v1/managed-schema

```diff
@@ -543,6 +548,7 @@
   <field name="providers" type="strings" required="false"/>
   <field name="publication_date" type="pdate" docValues="true" indexed="true" stored="false" useDocValuesAsStored="true"/>
   <field name="publication_year" type="pint" uninvertible="true" docValues="true" indexed="true" stored="true"/>
+  <field name="published" type="pdates"/>
```

Comment:
Have you added this field by yourself? Is it a necessary field? It seems not. If that is the case, delete it.

Reply:
It was already in the schema which I copied. I removed it.